encounters/README.md

```diff
@@ -0,0 +1,201 @@
+|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| id \[@root]                                      | A random UUID generated by the Adaptor                                                                                    |
+| code \[@code]                                    | `Encounter.type[0].coding[0].code` <sup>3</sup>                                                                           |
+| code \[@displayName]                             | `Encounter.type[0].coding[0].display` <sup>3</sup>                                                                        |
+| code \[@codeSystem]                              | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                          |
+| code / originalText                              | `Encounter.type[0].text` or else `Encounter.type[0].coding[0].display` <sup>4</sup>                                       |
+| statusCode                                       | fixed value =`"COMPLETE"`                                                                                                 |
+| effectiveTime                                    | `Enounter.period.start` and `Encounter.period.end` (if present)                                                           |
+| availabilityTime \[@value]                       | `Encounter.period.start`                                                                                                  |
+| author / agentRef / id \[@root]                  | `Encounter.participant[index].individual` where `Encounter.participant[index].type` contains a `coding.code` of `"REC"`   | 
+| author / time \[@value]                          | `List.date` <sup>5</sup>                                                                                                  |
+| location / locatedEntity / code \[@code]         | fixed value = `"394730007"`                                                                                               |
+| location / locatedEntity / code \[@codeSystem]   | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                          |
+| location / locatedEntity / code \[@displayName]  | fixed value = `"Healthcare related organisation"`                                                                         |
+| location / locatedEntity / name                  | The `name` field of the [Location](../locations/README.md) resource referenced by `Encounter.location`                    |
+| Participant2 / AgentRef / id \[@root]            | `Encounter.participant[index].individual` where `Encounter.participant[index].type` contains a `coding.code` of `"PPRF"`  | 
+
+
+<details>
+    <summary>Example XML</summary>
+
+```
+<ehrComposition classCode=\"COMPOSITION\" moodCode=\"EVN\">
```

Comment:
Is the outer `<component typeCode="COMP">` important at all?

Reply:
Each component within an `EHRFolder` is wrapped with `component`, so I thought of it as part of that element and not `ehrComposition`.